js/signaling.js

```diff
@@ -650,16 +639,8 @@
 					// Request has been aborted. Ignore.
 				} else if (this.currentRoomToken) {
 					if (this.pullMessagesFails >= 3) {
-						if (OCA.SpreedMe.webrtc) {
-							// Force leaving the call in WebRTC; leaving the
-							// room indirectly runs
-							// signaling.leaveCurrentCall(), but if the
-							// signaling fails to leave the call no event will
-							// be triggered and the call will not be left from
-							// WebRTC point of view.
-							OCA.SpreedMe.webrtc.leaveCall();
-						}
-						OCA.SpreedMe.app.connection.leaveCurrentRoom();
+						this._trigger('pullMessagesStoppedOnFail');
```

Comment:
please add a console.log here for easier debugging

Reply:
Done.